tpv/core/util.py

```diff
@@ -4,8 +4,9 @@
 import requests
 
 
-def load_yaml_from_url_or_path(url_or_path: str):
-    yaml = ruamel.yaml.YAML(typ='safe')
+def load_yaml_from_url_or_path(url_or_path: str, round_trip: bool = False):
+    typ = "rt" if round_trip else "safe"
```

Comment:
Can we simplify this to make it `roundtrip` always? Is there a disadvantage?

Reply:
With `safe` the data loads as plain Python objects (e.g. `dict`s and `list`s) and changing the loader to `rt` makes them `CommentedMap`, `CommentedList`, etc. Since only the linter needs round trip it this felt safer to me not knowing the code very well, but if you don't think it would cause problems I'm happy to change it to always load round trip.

Reply:
Regarding a disadvantage, if all the data are copied from the loader Entity objects (and not by copying the ref to the loader object representation) then it's probably ok. If not, it could potentially increase memory usage slightly and I am not certain everything would merge the same way without digging into ruamel.yaml.

Reply:
If I remember correctly, the current implementation is unlikely to preserve ruamel objects when copying. However, the current implementation will be obsolete once we merge this: https://github.com/galaxyproject/total-perspective-vortex/pull/136

I'm guessing pydantic won't preserve ruamel either?

Reply:
The round trip formatter is implemented as well, this should be good to go.